src/WebJobs.Script.WebHost/Management/InstanceManager.cs

```diff
@@ -75,7 +77,9 @@ public bool StartAssignment(HostAssignmentContext assignmentContext)
                     assignmentContext.ApplyAppSettings();
                 }
 
-                // TODO: specialize
+                // set flags which will trigger specialization
```

Comment:
This coupled with the Startup class change I made is how placeholder/specialization is controlled.